frontend/src/Crossword.tsx

```diff
@@ -415,9 +410,9 @@ const Crossword: React.FC<CrosswordProps> = (props) => {
     canvas.onmousedown = handleMouseDown;
     canvas.onmousemove = handleMouseMove;
     canvas.onmouseup = handleMouseUp;
-    canvas.ontouchstart = handleTouchStart;
-    canvas.ontouchmove = handleTouchMove;
-    canvas.ontouchend = handleTouchEnd;
+    canvas.addEventListener("touchstart", handleTouchStart, false)
```

Comment:
What's the difference between this and the previous?

Reply:
The previous does not work with my laptop's touchscreen, and this one works.

Reply:
According to https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener it should default to false

Reply:
Removed the false in the end :)